tests/functional/test_libraries.py

```diff
@@ -113,12 +113,16 @@ def test_gevent_monkey(pyi_builder):
 
 
 @xfail(is_darwin, reason='Issue #1895.')
+@importorskip('tkinter')
+@importorskip('Tkinter')
```

Comment:
These changes are *not* related to the pull-request, please remove them from here.